Claunia.Encoding/Claunia.Encoding.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Hm, seems like Visual Studio changed this file to UTF-8 (that's a BOM that's been added there)